LICENSE.md

```diff
@@ -0,0 +1,159 @@
+# Creative Commons Attribution-NonCommercial 4.0 International
```

Comment:
Creative Commons Licenses should not be used on software
See Section `Can I apply a Creative Commons license to software?`
https://creativecommons.org/faq/#Can_I_use_a_Creative_Commons_license_for_software.3F